text/lang-detection/README.md

```diff
@@ -1,5 +1,17 @@
-# Language Detection
+# Language detection model
```

Comment:
I can't tell if we have a consistent capitalization standard for these headings. @CarloLucibello any thoughts?